### PR TITLE
run provider generator to sync changes from core

### PR DESCRIPTION
**Pull request by durandom**
ran the generator again with

```bash
bundle exec rails g provider <name> --path ../
```

these are the changes.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -9,7 +9,6 @@ begin
 rescue LoadError
 end
 
-require 'manageiq-providers-hawkular'
 require 'bundler/gem_tasks'
```
Comment (abonas) [inline Rakefile]: why is it ok to replace this dependency?

**Comment by abonas** [comment]
@durandom can you please explain what is this generator and why it should be rerun?

**Comment by durandom** [comment]
@abonas the generator from core is used to scaffold a new provider. From time to time there are  some refactorings and just running it again is an easy way to keep the providers consistent.

File: Rakefile (hunk 1 above)
Comment (durandom) [inline Rakefile]: because this is name of the gem and since its in the gemspec its autorequired

**Comment by Fryguy** [comment]
> because it has a strong assumption that the provider will be a gem available in the rubygems.org. Btw. currently it creates a badge with a broken link, if the gem is not available there.

This is reasonable. Can you open this as an issue in manageiq where the generator lives?

**Pull request merged**